plugins/powershell/help.md

```diff
@@ -11,6 +11,7 @@
 * For local Orchestrator execution, ensure connectivity to any network resources the script will use
 * For remote server execution, a PowerShell-enabled server and administrative credentials
 * When adding a password for the script, try to avoid special or escape characters such as single or double quotes
+* Round Robin is not supported
```

Comment:
This might better worded as 

Currently we do not support the use of round-robin DNS lookups